src/styles/index.css

```diff
@@ -0,0 +1,19 @@
+@import "./card.css";
+@import "./stars.css";
+
+* {
+  box-sizing: border-box;
+}
+
+:root {
+  --color: #00adcd;
+  --color-text: #060b35;
+  --color-text-code-price: #616070;
+  font-family: "Roboto", sans-serif;
+}
+
```

Comment:
`font-family` should be in `html` selector

Reply:
Understood, thank you